praw/models/reddit/subreddit.py

```diff
@@ -978,9 +978,7 @@ def submit_gallery(
                     "caption": image.get("caption", ""),
                     "outbound_url": image.get("outbound_url", ""),
                     "media_id": self._upload_media(
-                        image["image_path"],
-                        expected_mime_prefix="image",
-                        gallery=True,
+                        image["image_path"], expected_mime_prefix="image", gallery=True,
```

Comment:
This change is the one that caused the merge conflict. If you change it back to what it was before, there should be no conflict.

```suggestion
                        image["image_path"],
                        expected_mime_prefix="image",
                        gallery=True,
```